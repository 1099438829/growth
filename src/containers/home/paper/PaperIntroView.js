import React, { Component } from 'react';
import { Text, View } from 'react-native';

class PaperIntroView extends Component {
  static componentName = 'PaperIntroView';

  render() {
    return (
      <View>
        <Text> Growth 纸质版 </Text>
      </View>
    );
  }
}

export default PaperIntroView;
