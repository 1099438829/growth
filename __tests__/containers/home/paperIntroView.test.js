import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import PaperIntroView from '../../../src/containers/home/paper/PaperIntroView';

it('renders correctly', () => {
  const tree = renderer.create(
    <PaperIntroView skillId={1} />,
  );

  const treeJson = tree.toJSON();
  expect(treeJson).toMatchSnapshot();
});
