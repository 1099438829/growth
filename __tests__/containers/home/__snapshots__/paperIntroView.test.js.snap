// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
     Growth 纸质版 
  </Text>
</View>
`;
